wp_api/src/request.rs

```diff
@@ -15,7 +18,6 @@ pub mod endpoint;
 
 const CONTENT_TYPE_JSON: &str = "application/json";
 const CONTENT_TYPE_MULTIPART: &str = "multipart/form-data";
-const LINK_HEADER_KEY: &str = "Link";
```

Comment:
We can use constants from `http::header`, which will avoid cluttering up this file with a bunch of extras (I'm using two more header names in this PR, and figured I'd align the older code while I was at it)